core/src/main/java/org/openstack4j/openstack/heat/domain/HeatAdoptStackData.java

```diff
@@ -0,0 +1,123 @@
+package org.openstack4j.openstack.heat.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+import org.openstack4j.model.heat.AdoptStackData;
+
+/**
+ * This class contains all elements required for the creation of <code>adopt_stack_data</code> element. It is used for stack adoption and as a return value for stack abandoning.
+ * It uses Jackson annotation for (de)serialization into JSON.
+ * 
+ * @author Ales Kemr
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HeatAdoptStackData implements AdoptStackData {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("action")
+    private String action;
+    
+    @JsonProperty("id")
+    private String id;
+    
+    @JsonProperty("name")
+    private String name;
+    
+    @JsonProperty("status")
+    private String status;
+    
+    @JsonProperty("template")
+    private Map<String,Object> template;
+    
+    @JsonProperty("resources")
+    private Map<String, Map<String, Object>> resources;
+
+    @Override
+    public String getAction() {
+        return action;
+    }
+    
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String getStatus() {
+        return status;
+    }
+
+    @Override
+    public Map<String,Object> getTemplate() {
+        return template;
+    }
+
+    @Override
+    public Map<String, Map<String, Object>> getResources() {
+        return resources;
+    }
+
+    @Override
+    public String toString() {
+        return "HeatAdoptStackData{" + "id=" + id + ", name=" + name + ", status=" + status + ", resources=" + resources + '}';
```

Comment:
I'd prefer
```
public String toString() {
MoreObjects.toStringHelper(this).omitNullValues()
  .add("id", id)
  .add(...)
  .toString();
}
```
What do you think @akemr?

Reply:
Sure, no problem. My simple toString was just generated by IDE. 
@auhlig Should I do the change in my branch and add the commit to this pull request?